src/fluree/db/query/exec.cljc

```diff
@@ -38,7 +38,7 @@
   containing a single result to the output channel instead of the single result
   alone."
   [q result-ch]
-  (if (:select-one q)
+  (if (get q "select-one")
```

Comment:
https://github.com/fluree/db/blob/f04ec1ce72520f5a5f58a1b9ada61fe89f510020/src/fluree/db/query/fql/parse.cljc#L432-L434

It could probably convert those to keywords at this stage as it's post-validation. The line of where to do that got a little fuzzy w/ queries, and I was operating under what I believed was consensus to push strings keys deeper into the code in the long run.

Reply:
It does that now ^^^ (in https://github.com/fluree/db/pull/438/commits/493556e536c66d1285d4248af1dbb59c20fcbe54)